Vizzuality/trase
# Update text and add link with anchor. Add auto anchors in about

## Asana

https://app.asana.com/0/0/1200047516960739/f

## Description

- Add auto links to headers in markdown in the About section
- Update text on the version modal
- Add anchor to link

## Testing instructions

Go to version modal in tool with the feature tag enabled
Test links. The last one should scroll to context versions
